src/model.jl

```diff
@@ -598,12 +627,86 @@ function read_geography(filepath=joinpath(@__DIR__,"..","input","world","The_Wor
   end
 
 
+  """
+  calc_area()
+  
+  Calcualtes the fractional area depending on the latitude in millions of square kilometers.
+  Total surface are of the earth is set to 510.000.000 km^2.
+  """ 
+function calc_area(mesh) 
+  @unpack nx,ny,area = mesh
+
+  area_frac = zeros(Float64,ny)
+  total_surface_area = 510    # approximate square kilometers of the earth's surface in millions
+  for i in 2:ny-1
+    area_frac[i] = total_surface_area * area[i]
+  end
+  area_frac[1] = total_surface_area * area[1]/nx
+  area_frac[ny] = total_surface_area * area[ny]/nx
+  return area_frac
+end
+
+
+"""
+calc_init_geography()
+
+Deletes all sea ice cells from the launch geography except for those at the North Pole.
+""" 
```

Comment:
They are not modified at the north pole, because the sea ice growth can only spread near sea ice or snow cells. As there are now snow cells in the input geopraphy in the Northern Hemisphere, there won't be any sea ice growth if the sea ice at the pole would be also deleted.
Later in the calculations we then always have (total monthly sea ice extent in km^2) - (sea ice at the north pole in km^2) = monthly sea ice growth which needs to be added in the geography matrix.  This way the sea ice area at the pole isn't added on top. 
I will add this explanation to the docstring.